### ISSUE TITLE
Meta: open todo or "help wanted" issues

### ISSUE THREAD

**Issue by tgross35** [davidlattimore/wild#14]
Hey! This is an awesome project :)

At this time, it isn't exactly clear what:

- Is done
- Is working but needs improvement
- Has a design but needs implementation
- Needs a design

Would you be able to put some of these things into separate issues with a "help wanted" label where relevant? This would make it easier for contributors to jump in.

Also, could you enable the discussions tab? Not super relevant now, but it's nice to separate problems from questions/comments as the project scales.

**Comment by davidlattimore** [comment]
Hey!

You're right that those things aren't currently documented. One concern I have, is that if I document all that, it can quickly fall out-of-date, or require quite a bit of work to keep up-to-date. But I guess it's a trade-off and perhaps I'm too far at the end of not documenting any of it - which has worked well while it's been mostly just me working on stuff - but perhaps should change as others start to contribute or want to contribute.

Writing up some issues and tagging them with good-first-issue seems like a good step. It can be tricky to find suitable issues though because they need to be something that doesn't need to be done right away. I've filed one so far, but will add more as I think of them.

But if you or anyone else is keen to try to contribute, I'm always up for discussing the project via a video call (see calendar link from my github to find a time), or if you don't like video calls, ping me on the rust-lang Zulip, send me an email or file issues or discussions (thanks for the suggestion, I've just enabled them). I've just updated the contributing section in the README with this info. I also added a what's-done section.

The big things that aren't yet done, at least those that I'm aware of are listed in the README. e.g. debug info being the next big one.

Currently I'm working on fixing bugs with what's there. Today I'm working on fixing some bugs related to thread-local variables in shared objects. I'm working towards making it possible to link rustc using Wild.

**Comment by tgross35** [comment]
Thanks for the update! Oh yeah totally understood that it's still in the bringup phase, but just having an idea of what you are working on vs. what is lower priority for you personally (so possibly open for somebody else) is awesome. Much apprecaited 👍

**Comment by mati865** [comment]
There are some "good first issues" now if you wanna give it a try.

**Comment by tgross35** [comment]
Indeed, this project is now fully up and running so I don't think this is needed anymore.
